### PR TITLE
DF2016 what what

### PR DESCRIPTION
**Pull request by barfait**
bringing this page kicking and screaming into the present

### CODE REVIEW

File: df16/index.html [hunk 1]
```diff
@@ -0,0 +1,236 @@
...
+				padding: 20px;
+			}
+
+			.challenge h3 {
+				margin-top: 0px;
+			}
+		</style>
+	</head>
+	<body>
+		<div class="row card" id="click_to_register">
```
Comment (pcon) [inline df16/index.html:59]: Need to change this to

`id="save_some_clicks"`

File: df16/index.html [hunk 2]
```diff
@@ -0,0 +1,236 @@
...
+						<li>No code allowed, this includes any of the programatic features such as Apex, Visualforce, custom JavaScript buttons, or integration with external systems.</li>
+						<li>Basic details related to travel plan should be captured in the Travel Request Object like Travel Request Name, Purpose of Travel, Travel Start Date, Days of travel (should be auto populated), Travel End Date.</li>
+						<li>Travel Request should also capture expenditure estimates like Airfare, Accommodation and Miscellaneous. There should a total field that calculates the estimates automatically.</li>
+						<li>If the estimate goes beyond $700, request should be auto submitted for approval from Employee’s manager.</li>
+						<li>Record should be locked when under approval process.</li>
+					</ol>
+				</div>
+			</div>
+		</div>
+		<div class="row card" id="automation_without_code">
```
Comment (pcon) [inline df16/index.html:82]: Change to

`id="allinone_system"`

File: df16/index.html [hunk 3]
```diff
@@ -0,0 +1,236 @@
...
+					<ol>
+						<li>This challenge requires a <a href="https://sfdc.co/df16devsignup">Force.com Developer Edition.</a></li>
+						<li>No code allowed, this includes any of the programatic features such as Apex, Visualforce, custom JavaScript buttons, or integration with external systems.</li>
+						<li><a href="http://bit.ly/externaldb">Use this as your external Database system.</a></li>
+						<li>You should link the Healthcare policy record to child customer records using object relationships as defined in the help doc <a href="https://sfdc.co/df16helpdoc">available here.</a></li>
+					</ol>
+				</div>
+			</div>
+		</div>
+		<div class="row card" id="integration_without_code">
```
Comment (pcon) [inline df16/index.html:103]: Change to

`id="redesign_visualforce"`

File: df16/index.html [hunk 4]
```diff
@@ -0,0 +1,236 @@
...
+					<h3>Requirements</h3>
+					<ol>
+						<li>This challenge requires a <a href="https://sfdc.co/df16devsignup">Force.com Developer Edition.</a></li>
+						<li>Refer the code in <a href="http://bit.ly/df16sldshack">this blog post.</a></li>
+						<li>The Visualforce page in the blog is designed to create an Account record so be sure you make appropriate code changes so that the new page creates a new Contact record.</li>
+					</ol>
+				</div>
+			</div>
+		</div>
+		<div class="row card" id="streaming">
```
Comment (pcon) [inline df16/index.html:123]: Change to

`id="lightning_contacts"`

File: df16/index.html [hunk 5]
```diff
@@ -0,0 +1,236 @@
...
+					<ol>
+						<li>This challenge requires a <a href="https://sfdc.co/df16devsignup">Force.com Developer Edition.</a></li>
+						<li>Create a Lightning Application named SearchContacts. The application is made of Lightning Components. You can choose to have one or more components depending on your design.</li>
+						<li>The application should have one search box where a user can enter an Account Name and the list below should return related contacts of the searched Account.</li>
+					</ol>
+				</div>
+			</div>
+		</div>
+
+		<div class="row card" id="lightning_components">
```
Comment (pcon) [inline df16/index.html:144]: Change to

`id="heroku"`

File: df16/index.html [hunk 6]
```diff
@@ -0,0 +1,236 @@
...
+						<li>Refer to <a href="bit.ly/df16herokuhack">this blog</a> for instructions on installing, running the app locally and deploying to Heroku.</li>
+						<li>Modify the app such that it can display Accounts from Salesforce.</li>
+						<li>Redeploy the changes on Heroku.</li>
+					</ol>
+					<h3>Clue</h3>
+					<p>You can refer to the <a href="https://sfdc.co/df16hackclue">Trailhead Heroku Connect module</a> for help.</p>
+				</div>
+			</div>
+		</div>
+		<div class="row card" id="heroku">
```
Comment (pcon) [inline df16/index.html:166]: Change to

`id="all_about_apis"``

File: df16/index.html [hunk 7]
```diff
@@ -0,0 +1,236 @@
...
+						<li>This challenge requires a <a href="https://sfdc.co/df16devsignup">Force.com Developer Edition.</a></li>
+						<li>Create an Apex REST Resource in Salesforce such that you can access and modify Salesforce data through external system(that external system for this hack can be your machine terminal).</li>
+						<li>Refer to our <a href="https://sfdc.co/df16restapihack">Apex Developer Guide</a> for reference and sample code.</li>
+						<li>Update the Rest Resource such that you are able to update a Contact’s description field as “Inactive Contact”.</li>
+						<li>You can use the sessionID for authentication instead of your username/password as shown in the sample code and demonstrate your solution by making the REST call to your ORG.</li>
+					</ol>
+				</div>
+			</div>
+		</div>
+		<div class="row card" id="analytics_api_and_d3">
```
Comment (pcon) [inline df16/index.html:188]: Change to

`id="windows"`

**Review by pcon — CHANGES_REQUESTED** [review]
Need to change the Ids to match the cards.

File: df16/index.html (hunk 7 above)
Comment (barfait) [reply df16/index.html:188]: no

**Comment by pcon** [comment]
Duplicate PR #9

**Pull request closed without merging**